revspeech/src/revspeech.cpp

```diff
@@ -75,9 +76,14 @@ static bool NarratorUnload(void)
 
 ///////////////////////////////////////////////////////////////////////////////
 
+extern "C" void getXtable();
+
 void revSpeechLoad(char *args[], int nargs, char **retstring,
 					Bool *pass, Bool *error)
 {
+	// Force a reference to the getXtable function
+	getXtable();
```

Comment:
This might be a bad idea - getXtable() takes parameters which are pointers to return values in...